cloudapi/config.go

```diff
@@ -161,9 +164,10 @@ func NewConfig() Config {
 		MetricPushInterval:    types.NewNullDuration(1*time.Second, false),
 		MetricPushConcurrency: null.NewInt(1, false),
 
-		TracesEnabled:      null.NewBool(false, false),
-		TracesHost:         null.NewString("insights.k6.io:4443", false),
-		TracesPushInterval: types.NewNullDuration(1*time.Second, false),
+		TracesEnabled:         null.NewBool(false, false),
+		TracesHost:            null.NewString("insights.k6.io:4443", false),
+		TracesPushInterval:    types.NewNullDuration(10*time.Second, false),
```

Comment:
Out of curiosity, is it intentional? What is the rational for expanding it?

Reply:
+1, I'd still do it every second

Reply:
As we have a longer timeout, I though about extending the interval. I restored the second; thanks!